_episodes/53.md

```diff
@@ -0,0 +1,169 @@
+
+* Faster filtering on columns containing long strings in Parquet data.
+* Predicate pushdown for real and double columns in MongoDB.
+* Support for Iceberg REST catalog in the `register_table` and `unregister_table` procedures.
+* Support for BEARER authentication for Nessie catalog.
+
+[Trino 433](https://trino.io/docs/current/release/release-433.html)
+
+* Improved support for Hive schema evolution.
+* Add support for altering table comments in the Glue catalog.
+
+Also note that Trino 433 also includes documentation for `CREATE/DROP CATALOG`.
+Check out the third SQL training session for a demo.
+
+## SQL routine competition
+
+Trino 431 finally delivered the long-awaited support for SQL routines. To
+celebrate and see what you all come up with, we are running a competition.
+[Share your best SQL routine]({% post_url 2023-11-09-routines %}), and win a
+reward sponsored by [Starburst](https://starburst.io).
+
+## Call for Java 21 testing
+
+<img src="{{site.baseurl}}/assets/images/logos/java-duke-21.png" width="100px" align="right">
+
+Java 21, the latest LTS release of Java, arrived in September 2023, and we want
+to take advantage of the performance improvements, language features, and new
+libraries. But to do so, [we need your input and confirmation that everything
+works as expected]({% post_url 2023-11-03-java-21 %}).
+
+## Concept of the episode: JDBC driver
+
+<img src="{{site.baseurl}}/assets/images/logos/jdbc-small.png" width="100px" align="right">
+
+[Java Database Connectivity
+(JDBC)](https://en.wikipedia.org/wiki/Java_Database_Connectivity) is an
+important standard for any JVM-based application, that wants to access a
+relational database. Trino ships a JDBC driver that abstracts all the low-level
+details of our conversational REST API for client tools and supports various
+authentication mechanisms, TLS, and other features. This allows tools like
+Coginiti to ignore those details, and work with the community on any
+improvements for the benefit of all users.
+
+## Client tool focus on Coginiti
+
+Matthew and Roman are joining us from [Coginiti](https://www.coginiti.co).
+Coginiti delivers higher-quality analytics faster. Coginiti provides an
+AI-enabled enterprise data workspace that integrates modular development,
+version control, and data quality testing throughout the analytic development
+lifecycle.
+
+<a href="https://www.coginiti.co">
+  <img src="{{site.baseurl}}/assets/images/logos/coginiti-small.png">
+</a>
+
+With the support for Trino, Coginiti as a client tool provides access to all the
+configured catalogs in Trino. It enables data engineers and analyst to work
+together in a shared platform, reducing duplication in their work, and bringing
+"Don't repeat yourself (DRY)" to analysts.
+
+We talk about why Coginiti added [support for
+Trino](https://www.coginiti.co/databases/trino/). Coginiti is not compute
+platform itself, but access to many platforms enables a data blender thinking.
```

Comment:
This sentence kinda just reads weird. Not totally sure what you mean by "A data blender thinking"